raspbian/applications/camera.md

```diff
@@ -269,6 +269,18 @@ Displays the exposure, analogue and digital gains, and AWB settings used.
 
 Sets blue and red gains (as floating point numbers) to be applied when `-awb -off` is set e.g. -awbg 1.5,1.2
 
+```
+--analoggain,	-ag
+```
+
+Sets the analog gain value directly on the sensor (floating point value from 1.0 to 8.0 for the OV5647 sensor on the version 1 camera board, and 1.0 to 12.0 for the IMX219 sensor on the version 2 board).
+
+```
+--digitalgain,	-ag
```

Comment:
Should the short-option here be `-dg`, since `-ag` is already used by `--analoggain` ?

Reply:
Yup. Will fix.